server/src/lib/model/db.ts

```diff
@@ -194,26 +323,86 @@ export default class DB {
         SELECT clips.*
         FROM clips
         LEFT JOIN sentences on clips.original_sentence_id = sentences.id
-        WHERE is_valid IS NULL AND clips.locale_id = ? AND client_id <> ? AND
-              NOT EXISTS(
-                SELECT *
-                FROM votes
-                WHERE votes.clip_id = clips.id AND client_id = ?
-              )
+        WHERE is_valid IS NULL AND clips.locale_id = ? AND client_id <> ?
+        AND NOT EXISTS(
+            SELECT *
+            FROM votes
+            WHERE votes.clip_id = clips.id AND client_id = ?
+          )
+        AND (sentences.clips_count <= 10 OR sentences.clips_count IS NULL)
```

Comment:
I added this filter here because I noticed during testing that finding clips was pretty slow if you're a user who has already voted on a lot of things (1-2s when you hit the db directly), and this very quickly narrows the field of clips, especially since all of our other logic already prioritizes clips with low clip_counts and the likelihood of a sentences increasing past that is minimal. 

There are some sentences that have a clip count above 10 where none of the clips are valid, but they tend to be super old clips from before that other logic was implemented. It'll be moot soon anyway once we implement single-sentence record limits.